components/header/index.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
```

Comment:
creation of Header component is not part of this issue.
Let's merge the changes done for the Header component

Reply:
I created it so that i can show the storybook sample as mentioned in the issue.
Will wait until @aryamantodkar pr is merged